backend/api_handler.py

```diff
@@ -135,21 +137,25 @@ def handler(event, context):
             print(f'Error managing groups due to: {e}')
             groups = []
 
+        set_context(RequestContext(ENGINE, username, groups, ES))
+
         app_context = {
```

Comment:
[Question] - this app_context can be replaced later in graphql_sync by get_context right?

Reply:
That's the goal)